ansible/roles/test/files/ptftests/fib_test.py

```diff
@@ -112,6 +112,9 @@ def setUp(self):
          - single_fib_for_duts:   have a single fib file for all DUTs in multi-dut case. Default: False
         '''
         self.dataplane = ptf.dataplane_instance
+        self.asic_type = self.test_params.get('asic_type')
+        if self.asic_type == "marvell":
```

Comment:
I remember that the asic type been changed to `marvell-armhf` sinice 202205, should we use:
```
if "marvell" in self.asic_type
```

Reply:
Hi @Blueve 

   I have checked the ansible facts gathered by the code. facts  asic_type is marvell and asic_subtype is marvell-armf. should we use asic_subtype in place of asic_type?
   
   Please find details given below:-

   ansible_facts": {"discovered_interpreter_python": "/usr/bin/python", "dut_basic_facts": {"asic_subtype": "marvell-armhf", "asic_type": "marvell", "branch": "202205", "build_date": "Thu Nov 24 12:44:20 UTC 2022", "build_number": 179361, "build_version": "202205.179361-7c1015b45
 
 Thanks
Rajneesh